### DISCUSSION TITLE
pag动画颜色在iOS设备上发生了左右断层

### DISCUSSION THREAD

**Discussion by MrYu4** [Tencent/libpag#1744] [Q&A] [answered]
【版本信息】
ios(4.2.6)
【平台信息】
iOS原生
【预期的表现】
左右颜色过渡自然

【实际的情况】
画面左半部分显示一个色块
![IMG_317103B6811F-1](https://github.com/Tencent/libpag/assets/22674880/d098ef4b-db7e-4b2e-baf8-7b8a1d4d993e)

【Demo及附件】
[tmpAEtoAMEProject-沙丘pag特效_1文件夹.zip](https://github.com/Tencent/libpag/files/12682414/tmpAEtoAMEProject-.pag._1.zip)

**Comment by MrYu4** [comment]
😑

**Reply by domchen** [reply, marked as answer]
最新版本桌面预览工具里替换图片是正常的，文件没什么问题。移动端是和桌面同一套代码。你在IOS用的4.2.6版本SDK我们没有发布过，看看是不是写错了版本号，可能是版本不对或者过旧有新特效没有支持，升级一下到最新的SDK版本就行了。
